server/tests/audit_math/test_supersimple.py

```diff
@@ -60,6 +60,62 @@ def test_compute_diluted_margin(contests):
         )
 
 
+def test_find_no_discrepancies(contests, cvrs):
+
+    # Test no discrepancies
+    sample_cvr = {
+        0: {
+            "Contest A": {"winner": 1, "loser": 0},
+            "Contest B": {"winner": 1, "loser": 0},
+            "Contest C": {"winner": 1, "loser": 0},
+            "Contest D": {"winner": 1, "loser": 0},
+            "Contest E": {"winner": 1, "loser": 0},
+        }
+    }
+
+    for contest in contests:
+        discrepancies = supersimple.compute_discrepancies(
+            contests[contest], cvrs, sample_cvr
+        )
+
+        assert discrepancies[0]["counted_as"] == 0
+        assert discrepancies[0]["weighted_error"] == 0
+        assert discrepancies[0]["cvr"] == discrepancies[0]["sample_cvr"]
+
+
+def test_find_one_discrepancy(contests, cvrs):
+
+    # Test no discrepancies
```

Comment:
this comment doesn't apply. maybe copy-paste error?

Reply:
Yep :sweat_smile: